sdk/openai/openai/samples/cookbook/bringYourOwnData/index.js

```diff
@@ -0,0 +1,124 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+// Importing the @azure/search-documents library
+const { SearchIndexClient, SearchClient, AzureKeyCredential, odata } = require("@azure/search-documents");
+
+// Importing the @azure/openai library
+const { OpenAIClient } = require("@azure/openai");
+
+// Importing the index definition and sample data
+const hotelData = require('./hotels.json');
```

Comment:
Instead of requiring a JSON file, could we just use the File APIs to load it?  I'd like to see a solution that works for both import and require.

Reply:
Updated with [`90f9611`](https://github.com/Azure/azure-sdk-for-js/pull/27298/commits/90f9611a8f53cd98e2ca46f12c1bfbd931261293)